### ISSUE TITLE
RFi Module does not work properly...

### ISSUE THREAD

**Issue by the-Blue-Falcon** [theInfectedDrake/TIDoS-Framework#14]
The RFi module under `VulnLysis/SerioBugs` doesn't seem to work properly. I have a website which is indeed vulnerable to RFi. But TIDoS wasn't able to detect it.
```

 Type Selected : RFI

   ===========================================
    R E M O T E   F I L E   I N C L U S I O N
   ===========================================

    Choose from the options:
    [1] Custom Targetting
    [2] Automated Scanning

 [#] Enter your choice :> 1
 [#] Enter the point scope parameter :> /file.php?url=
 [!] Your input has a "/" in the beginning,..
 [#] Do you mean root directory? (y/n) :> y
 [+] URL : http://site.com//file.php?url=
 [+] Url successfully parsed!
 [*] Trying basic fetch...
 [+] Payload : http://www.google.com
 [*] Fetching http://site.com//file.php?url=http://www.google.com
 [!] Heuristics reveal that the page may not be vulnerable to RFI!
 [*] Trying null byte character injection...
 [+] Payload : http://www.google.com%00
 [*] Fetching http://site.com//file.php?url=https://google.com%00
 [-] This RFI module could not find out any RFI.
 [-] This module is extreme basic (more improvements on the way).
 [+] Done!
```
I think this is due to the improper handling of the file parameter. Please fix this.

**Comment by theInfectedDrake** [comment]
I just got that fixed in e56a319. Can you confirm that?

**Comment by the-Blue-Falcon** [comment]
Yep, I can! Works as a charm 💯. You are fast as lightning!
